pkg/resourcevalidator/sp_description.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcevalidator
+
+import (
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// ShadowPodDescription is a struct that contains the main informations about a shadow pod.
+type ShadowPodDescription struct {
+	Name      string
```

Comment:
Name shall be the combination of both namespace and name, otherwise it is not guaranteed to be unique.

Reply:
so i have to use NamspacedName?

Reply:
Yes, that could be a solution